public/data/Abdus_Salam.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Abdus Salam",
+  "type": "personal",
+  "bio": "Hi there! I am Python Developer and doing my best to contribute to open source. I'm not limited to Python only, I can do more fun things! (it's a long list) Connect with me and find out",
+  "avatar": "https://github.com/abdussalam02.png",
+  "links": [
+    {
+        "name": "Check out my personal website",
+        "url": "https://abdussalam.vercel.app",
+        "icon": "globe"
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/abdussalam02",
+      "icon": "github"
+    },
+    {
+        "name": "I'm on Telegram most of my time",
+        "url": "t.me/aladdin_02",
+        "icon": "Telegram"
```

afaik this should work since the PR was merged, but since there are comments related to this, let us try

```suggestion
        "icon": "telegram"
```

---

Just to let you know, I've also created a hotfix for this https://github.com/EddieHubCommunity/LinkFree/pull/851

---

> Just to let you know, I've also created a hotfix for this #851

That would have been a good teaching opportunity for you/learning opportunity for @abdussalam02